### DISCUSSION TITLE
Дата создания проекта/команды и включения в состав команды/проекта

### DISCUSSION THREAD

**Discussion by gfg7** [habralab/garnet-team#78] [Идеи]
Отображать дату создания команды/проекта и принятия в состав участника в команду, команды в проект
Позволит пользователям дать поверхностную оценку по внесенному вкладу и опыту совместной работы в таком составе:
- Когда был сформирован проект/команда - как давно работают над этой идеей, когда владелец решил собрать команду?
- Когда участник был включен в команду - новичок в команде или держится с самого основания?
- Когда команда была включена в проект - команда была с самого старта проекта или присоединились совсем недавно?

**Comment by Demorey** [comment]
Поддерживаю, даты создания/вступления нужно добавить, это будет не только полезно для пользователей, но и в дальнейшем можно будет использовать для расчета рейтинга пользователей/команд.

**Comment by picolino** [comment]
Даты создания команды\проекта можно теперь добавить через аудитную информацию (как сделано в пользователях)
